app/tests/integration/conftest.py

```diff
@@ -108,20 +156,15 @@ def fill_timeline_page(
     timeline.proposed_start_date = today
     timeline.proposed_experiment_duration = "25"
     timeline.population_precentage = "100.0"
-    if request.node.get_closest_marker("use_variables"):
-        timeline.firefox_channel = f"{variables[experiment_type]['channels']}"
-        timeline.firefox_min_version = f"{variables[experiment_type]['min_version']}"
-        timeline.firefox_max_version = f"{variables[experiment_type]['max_version']}"
-    else:
-        timeline.firefox_channel = "nightly"
-        timeline.firefox_min_version = "75"
-        timeline.firefox_max_version = "100"
+    timeline.firefox_channel = default_data.channels
```

Comment:
The `firefox_channel` attribute suggests that it expects a single channel, but here plural `channels` is assigned as a value. Is this correct? Can you update the variables names to clarify what's happening?

Reply:
So this has to be plural because there technically could be multiple channels. In this context we aren't doing that but it is a possibility in the future